diffrax/misc/sde_kl_divergence.py

```diff
@@ -23,7 +26,7 @@ class _AugDrift(eqx.Module):
     def __call__(self, t, y, args):
         y, _ = y
         context = self.context(t)
-        aug_y = jnp.concatenate([y, context], axis=-1)
+        aug_y = jnp.concatenate([y, context], axis=-1) if context is not None else y
```

Comment:
Nit: flipping the `if` and `else` branches allows for switching `if context is not None` down toj ust `if context is None`.